yarn.lock

```diff
@@ -5105,6 +5105,21 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@octokit/core@npm:^5.0.0":
```

Comment:
Looks like we are ending up with duplicate versions. Do we need to combine these? or do we have some mismatch in versions somewhere?

Reply:
I am not sure why yarn didn't merge those,  fixed.